src/ORM/Connect/DBConnector.php

```diff
@@ -66,6 +66,23 @@ protected function databaseError($msg, $errorLevel = E_USER_ERROR, $sql = null,
         }
     }
 
+    protected function duplicateEntryError(
```

Comment:
Protected here so other DB implementations (e.g. postgres) can use this if someone wants to have the same UX with non-MySQL

Reply:
The method name says error, though it throws an exception.

Probably just rename it to `throwDuplicateEntryException` - seems unlikely any subclass wouldn't want to throw that exception

Reply:
The other two related methods say `error` at the end. If I'm in the IDE and I want to get one of those methods, I might type `$this->error` and let the IDE show me what's available. If this method has `exception` instead of `error` then it's not going to appear and I may not be aware it's available.

I'd prefer to leave it as is for consistency and DX.

Reply:
OK makes sense to leave as is, probably not worth the effort to change every 'error' to 'exception' given there are subclasses involved.